docs/devel/hostpath-provisioner.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hostpath-provisioner
+  labels:
+    app: hostpath-provisioner
+spec:
+  selector:
+    matchLabels:
+      app: hostpath-provisioner
+  replicas: 1
+  revisionHistoryLimit: 0
+  template:
+    metadata:
+      labels:
+        app: hostpath-provisioner
+    spec:
+      serviceAccount: hostpath-sa
+      serviceAccountName: hostpath-sa
+      containers:
+        - name: hostpath-provisioner
+          image: mazdermind/hostpath-provisioner:latest
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: PV_DIR
+              value: /var/run/kubevirt
```

Comment:
@davidvossel I picked /var/run/kubevirt as I noticed we had some other PVs pointing there already, if there is a better location I would love to hear about it.

Reply:
a sub directory in /var/run/kubevirt might be best. Also one thing to note here, I think /var/run is ephemeral and is wiped between host restarts.

Reply:
The hostpath provisioner, creates sub directories for each PVC under whatever we have as the PV_DIR, is that sufficient or should I make /var/run/kubevirt/hostpathprovisioner?

Reply:
yep, we should us a subdirectory